UI node bundle comment correction

ickshonpe opened the pull request:
# Objective

Doc comment for the global_transform: GlobalTransform in NodeBundle says:

```
/// This field is automatically managed by the UI layout system.
```

The `GlobalTransform` component is the thing being managed, not the `global_transform` field, and the `TransformPropagate` systems do the managing, not the UI layout system.

rparrett reviewed (approved):
Checked the rendered docs. Looks good / makes sense to me.

While reviewing I noticed that the docs for the `transform` fields for these various bundles all make references to `NodeBundle`.

Weibye reviewed (approved):
I agree this is more correct :+1:

ickshonpe commented:
> Checked the rendered docs. Looks good / makes sense to me.
> 
> While reviewing I noticed that the docs for the `transform` fields for these various bundles all make references to `NodeBundle`.

Oh yep I see, better fix that too.

ickshonpe commented:
```/// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.```

Changed this line for the `Transform` comments so it names the specific bundle each time. 
For `GlobalTransform` I deleted it, as it seems odd to have the information there when you don't use `GlobalTransform` to change positions.

rparrett commented on crates/bevy_ui/src/node_bundles.rs:
```diff
@@ -75,7 +75,7 @@ impl Default for NodeBundle {
     ///
-    /// This field is automatically managed by the UI layout system.
+    /// This component is automatically managed by the UI layout system.
     /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
```
There's also a copy of this comment here, associated with the `node` field, with the same copy/paste mistake.

Seems safe to get rid of this one too to me. Not every bundle has it. But maybe it was intended that it say "to alter the **size**?

alice-i-cecile commented:
@ickshonpe once merge conflicts are cleaned up, I'll merge this.
